src/components/Loader/Loader.js

```diff
@@ -0,0 +1,14 @@
+import './Loader.css'
+import React from 'react'
+
+function Loader() {
+  return (
+    <div className="lds-facebook">
```

Comment:
non capisco molto il nome classe o cosa c'entri con facebook

Reply:
è un component che ho preso da un pacchetto online, se serve lo ritocco in modo che funzioni con dei nomi più attinenti alla mia repo.